src/gt4py/gtc/python/gtir_to_pnir.py

```diff
@@ -8,23 +8,21 @@
 
 
 class GtirToPnir(NodeTranslator):
-    def visit_Computation(self, node: gtir.Computation) -> pnir.Stencil:
+    def visit_Stencil(self, node: gtir.Stencil) -> pnir.Stencil:
         stencil_obj = pnir.StencilObject(
             name=node.name, params=node.params, fields_metadata=node.fields_metadata
```

Comment:
@egparedes  e.g. here
```suggestion
            name=str(node.name), params=node.params, fields_metadata=node.fields_metadata
```